microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -496,6 +497,23 @@ def _configure_vdpa_devices(self, node: Node) -> None:
             node.tools[Chown].change_owner(file=PurePath(device_path), user=user)
             node.tools[Chmod].chmod(path=device_path, permission=permission, sudo=True)
 
+    def _set_data_disk(self) -> None:
+        datadisk_name = ""
+        disks = self.node.tools[Lsblk].get_disks()
+        # get the first unmounted disk (data disk)
+        for disk in disks:
+            if disk.is_mounted:
+                continue
+            if disk.name.startswith("sd"):
+                datadisk_name = disk.device_name
+                break
+        # running lsblk once again, just for human readable logs
+        self.node.execute("lsblk", shell=True)
+        if not datadisk_name:
+            raise LisaException("No data disk found")
+        self._log.debug(f"Using data disk: {datadisk_name}")
+        self.env_vars["DATADISK_NAME"] = datadisk_name
```

Comment:
Why set it to env variables? If it's used in the class only, return the name. Not use env or class level variables.

Reply:
This env variable is need by the ch community test scripts that are run in the CH LISA tests.